temporian/beam/operators/add_index.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Dict, List, Tuple, Optional, Iterable
+
+from collections import defaultdict
+import apache_beam as beam
+import numpy as np
+
+from temporian.core.operators.add_index import (
+    AddIndexOperator as CurrentOperator,
```

Comment:
why do this rename? the original symbol isn't used elsewhere

Reply:
Each such file handle a single operator.
There are some boilerplate code that only depends on the operator class.
By having a common class name, we make this boilerplate easier to read and type.